app/src/main/res/values/strings.xml

```diff
@@ -203,6 +203,7 @@
     <string name="look_and_feel_theme_color_woodland">Woodland</string>
     <string name="look_and_feel_use_custom_tabs">Use custom tabs</string>
     <string name="look_and_feel_use_private_tabs">Use private custom tabs if available</string>
+    <string name="look_and_feel_use_swipe_back">Use swipe to navigate back on posts</string>
```

Comment:
Name it something like Post gestures, and then have `none option` and `swipe back option`

Reply:
You can find some ideas for the nomenclature here https://github.com/dessalines/jerboa/pull/872/files

Reply:
I've implemented it as you suggested, I've titled the list `Post Navigation Gestures`, and provided a `Disabled` option and a `Swipe Left to Navigate Back` option.